crates/ty_python_semantic/src/types.rs

```diff
@@ -2127,12 +2199,12 @@ impl<'db> Type<'db> {
                 any_protocol_members_absent_or_disjoint(db, protocol, nominal, visitor)
             }),
 
-            (Type::ProtocolInstance(protocol), other)
-            | (other, Type::ProtocolInstance(protocol)) => visitor.visit((self, other), || {
+            (Type::ProtocolInstance(protocol), other_ty)
+            | (other_ty, Type::ProtocolInstance(protocol)) => visitor.visit((self, other), || {
```

Comment:
Although it's not directly related to this PR, I fixed a subtle bug. `other` was incorrectly shadowed here.